pkg/graph/resolvers/plan_discussion.go

```diff
@@ -133,10 +343,11 @@ func sendPlanDiscussionCreatedEmail(
 	emailBody, err := emailTemplate.GetExecutedBody(email.PlanDiscussionCreatedBodyContent{
 		ClientAddress:     emailService.GetConfig().GetClientAddress(),
 		DiscussionID:      planDiscussion.ID.String(),
-		UserName:          planDiscussion.CreatedByUserAccount(ctx).CommonName,
+		UserName:          createdByUserName,
 		DiscussionContent: planDiscussion.Content.RawContent.ToTemplate(),
 		ModelID:           modelPlan.ID.String(),
 		ModelName:         modelPlan.ModelName,
+		Role:              createdByUserRole, //TODO verify with Natasha
```

Comment:
Is this still `TODO`?

Reply:
Thanks! No, I verified about this, thanks for catching it!